docs/src/examples/DC-DC converter.jl

```diff
@@ -0,0 +1,141 @@
+
+# Definition of the dynamics functions $f_p$ of the system:
+b = SVector(vs/xL, 0.0);
+A1 = SMatrix{2,2}(-rL/xL, 0.0, 0.0, -1.0/xC/(r0+rC));
+A2 = SMatrix{2,2}(-(rL+r0*rC/(r0+rC))/xL, 5.0*r0/(r0+rC)/xC,
+    -r0/(r0+rC)/xL/5.0, -1.0/xC/(r0+rC));
+F_sys = let b = b, A1 = A1, A2 = A2
+    (x, u) -> u[1] == 1 ? A1*x + b : A2*x + b
+end;
+# Definition of the growth bound functions of $f_p$:
+ngrowthbound = 5;
+A2_abs = SMatrix{2,2}(-(rL+r0*rC/(r0+rC))/xL, 5.0*r0/(r0+rC)/xC,
+                      r0/(r0+rC)/xL/5.0, -1.0/xC/(r0+rC));
+L_growthbound = let A1 = A1, A2_abs = A2_abs
+    u -> u[1] == 1 ? A1 : A2_abs
+end;
+# Here it is considered that there is no system and measurement noise:
+sysnoise = SVector(0.0, 0.0);
+measnoise = SVector(0.0, 0.0);
+
+# Definition of the discretization time step parameters: `tstep` and `nsys`:
+tstep = 0.5;
+nsys = 5;
+
+# Finally, we build the control system:
+contsys = AB.NewControlSystemGrowthRK4(tstep, F_sys, L_growthbound, sysnoise,
+                                       measnoise, nsys, ngrowthbound);
+
+# ### Definition of the control problem
+# Definition of the state-space (limited to be rectangle):
+_X_ = AB.HyperRectangle(SVector(1.15, 5.45), SVector(1.55, 5.85));
+
+# Definition of the input-space, the later discretization of the input ensures that it can only take the values $1$ or $2$:
+_U_ = AB.HyperRectangle(SVector(1), SVector(2));
+
+# ### Definition of the abstraction
+
+# Definition of the grid of the state-space on which the abstraction is based (origin `x0` and state-space discretization `h`):
+x0 = SVector(0.0, 0.0);
+h = SVector(2.0/4.0e3, 2.0/4.0e3);
+Xgrid = AB.GridFree(x0, h);
+# Construction of the struct `DomainList` containing the feasible cells of the state-space.
+# Note, we used `AB.INNER` to make sure to add cells entirely contained in the domain because we are working with a safety problem.
+Xfull = AB.DomainList(Xgrid);
+AB.add_set!(Xfull, _X_, AB.INNER)
+
+# Definition of the grid of the input-space on which the abstraction is based (origin `u0` and input-space discretization `h`):
+u0 = SVector(1);
+h = SVector(1);
+Ugrid = AB.GridFree(u0, h);
+# Construction of the struct `DomainList` containing the quantized inputs:
+Ufull = AB.DomainList(Ugrid);
+AB.add_set!(Ufull, _U_, AB.OUTER);
+
+# Construction of the abstraction:
+symmodel = AB.NewSymbolicModelListList(Xfull, Ufull);
+@time AB.compute_symmodel_from_controlsystem!(symmodel, contsys)
+
+# ### Construction of the controller
+# In this problem, we consider both: the initial state-space and the safety state-space are equal to the entire state-space.
+#
+# Computation of the initial symbolic states:
+Xinit = AB.DomainList(Xgrid);
+union!(Xinit, Xfull)
+initlist = Int[]
+for pos in AB.enum_pos(Xinit)
+    push!(initlist, AB.get_state_by_xpos(symmodel, pos))
+end
+# Computation of the safety symbolic states:
+Xsafe = AB.DomainList(Xgrid)
+union!(Xsafe, Xfull)
+safelist = Int[]
```

Comment:
`safelist = [AB.get_state_by_xpos(symmodel, pos) for pos in AB.enum_pos(Xsafe)] `

Reply:
Indeed it's shorter :)